corehq/apps/export/export.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, writer):
         self._path = None
```

Comment:
decided to go this route because in the end i think it's less magical and it's how we handle other top level config properties like `transform_dates`. talked it over with ethan as well. at the end of the day i feel ok about it, but most importantly it's working!